packages/remark-lint-blockquote-indentation/readme.md

```diff
@@ -157,16 +157,16 @@ Due to this, it’s recommended to configure this rule with `2`.
 
 ##### `ok.md`
 
-When configured with `2`.
+When configured with `4`.
```

Comment:
@jablko these changes appear to unrelated to giving rules access to settings, and also doesn't appear to solve an issue with the docs, but moves things around?

Reply:
You're right: It's because JS objects aren't order preserving. Before, the examples were ordered by the numeric option value. Now they're in the order they're defined in the docblocks, which is either irrelevant or an improvement?

The reason for the change is that the examples were previously indexed by the option value alone. Now they're indexed by `{settings, options}`, because the behavior depends on `settings` --- at least in the case of remark-lint-emphasis-marker and remark-lint-strong-marker.